lib/colors.zsh

```diff
@@ -2,8 +2,10 @@ declare -Ag tw_color_mappings=(
   "foreground" "default"
   "primary" "magenta"
   "secondary" "blue"
+  "notice" "yellow"
```

Comment:
Also had a hard time coming up with a variable name for that! 😅 `notice` sound good though, it will probably also be used for other stuff in the future